### ISSUE TITLE
Complete the walking skeleton (locally)

### ISSUE THREAD

**Issue by bojeans** [bojeans/travel-blog#7]
this is where we will use a fetch request to hopefully return the data in the browser on localhost3000
before we start, have a look at react testing library to see if can figure out how to use TDD (make another ticket if so and to that first), double check if the same for nextJS. If unsuccessful re testing, can investigate after deployment during the next planning phase. 

Acceptance Criteria:
- data should render to the browser successfully from the backend on localhost3000
